forge-gui/res/cardsfolder/a/averna_the_chaos_bloom.txt

```diff
@@ -3,5 +3,5 @@ ManaCost:G U R
 Types:Legendary Creature Elemental Shaman
 PT:4/2
 R:Event$ Cascade | ValidPlayer$ You | ActiveZones$ Battlefield | ReplaceWith$ DBLand | ReplacementResult$ Updated | Description$ As you cascade, you may put a land card from among the exiled cards onto the battlefield tapped. (Do this after the last card is exiled but before deciding whether to cast a nonland card.)
-SVar:DBLand:DB$ ChangeZone | Hidden$ True | Origin$ Exile | ChooseFromDefined$ ReplacedCards.Land | ChangeNum$ 1 | Destination$ Battlefield | Tapped$ True
+SVar:DBLand:DB$ ChangeZone | Hidden$ True | Origin$ Exile | ChooseFromDefined$ ReplacedCards.Land | Destination$ Battlefield | Tapped$ True
```

Comment:
just find/replace removing it everywhere will break some cards lol

maybe the behaviour needs to be synchronized, then "search...any number" scripts don't need to use extra SVar

pretty sure we had an issue for that somewhere

Reply:
ugh yeah, i really need to add that special "anyNumber"